src/app/services/sms/sms.service.ts

```diff
@@ -248,7 +251,7 @@ const sendSms = (
         logType: LogType.success,
       })
 
-      return result
+      return result as true
```

Comment:
do we need to cast here? did it not infer the types automatically? Might want to debug why it is not if that's the case

(checked on my machine on my current react branch, it is automatically inferring)